server/src/main/resources/dst_server.conf

```diff
@@ -0,0 +1,8 @@
+# Here is the server port that Dst-Server listens on.
+DstServer.listeningPort = 10086
+
+# This configures whether the current process is the master in Dst partation.
+DstServer.isMaster = true
+
+# Change how many shards Dst-Server has
+DstServer.shardNum = 8
```

Comment:
1. The name of this file should be `dst.default.conf`
2. The structure of this file should be :
```yml
store {
  listeningPort: 10086

  isMaster: true
  
  shardNum: 8
}
```